.gitignore

```diff
@@ -1,3 +1,4 @@
 *manifest.json
 *.swp
 .idea
+amazon-eks-node-*.json
```

Comment:
you're trying to ignore the `version-info.json` right? can we just add `*version-info.json` since the `ami_name` might not always start with `amazon-eks-node`?